documentation/docs/steps/cloudFoundryCreateSpace.md

```diff
@@ -0,0 +1,34 @@
+# ${docGenStepName}
+
+## ${docGenDescription}
+
+## Prerequisites
+
+* You have a user for the SAP Cloud Platform Cloud Foundry Environment
+* Credentials have been configured in Jenkins with a dedicated Id
+
+## ${docGenParameters}
+
+## ${docGenConfiguration}
+
+## ${docJenkinsPluginDependencies}
+
+## Example
+
+* ### Single Space Creation in Cloud Foundry with a simple example
```

Comment:
Why the three # in a list?

Reply:
Because it is a sub heading of `Example`.

Reply:
But the `*`makes it into a list? That looks strange to me, but it works, I checked. I would still remove the `*` as it is strange to have a list of one... I will not insist, however :)

Reply:
Asterisk has already been removed